Clarify frontend upgrades (#3200)
chef/chef-server: docs-chef-io/content/server/upgrades.md

## Chef Infra Server 14 Upgrade Process

> ### Frontends in a Chef Backend HA
> 
> Skip the first two steps in the Standalone Server section, then proceed to install the package and upgrade. Also skip the final reindex step.
> Anything to do with the database in a Chef Backend cluster will happen on the backend cluster.

Reviewer: ```suggestion
{{< note >}}
For frontend nodes in a Chef Backend HA cluster, skip the first two steps in the [Standalone Server](#standalone-server) section.
{{< /note >}}
```

Reviewer 2: It is useful to clarify `Anything to do with the database in a Chef Backend cluster will happen on the backend cluster.` explicitly.
